src/main/java/com/solvd/micro9/users/web/WebConfig.java

```diff
@@ -43,4 +54,69 @@ public WebClient.Builder webClient() {
         return WebClient.builder();
     }
 
+    public GraphQLScalarType localDateTimeScalar() {
```

Comment:
@eyakauleva try to make more elegant solution here

Reply:
done 679c337